shakenfist/db.py

```diff
@@ -235,10 +235,18 @@ def create_snapshot(snapshot_uuid, device, instance_uuid, created):
             'uuid': snapshot_uuid,
             'device': device,
             'instance_uuid': instance_uuid,
-            'created': created
+            'created': created,
+            'status': 'creating'
         })
 
 
+def finalize_snapshot(snapshot_uuid, instance_uuid, created):
+    snapshot = etcd.get('snapshot', instance_uuid, created)
+    snapshot['status'] = 'ready'
```

Comment:
"ready" is a new name for a state. The current ones are: preflight; creating; created; deleting; deleted.

Do we think there is value in having a standard naming convention for states? I suspect it would reduce errors in the future?

Reply:
That seems perfectly reasonable to me